build/yamls/base/antctl.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Will antctl.yml be incorporated into antrea.yml or distributed separately? Did not see either with this PR.

Reply:
Thanks, I forgot to add it to the kustomize.